rotkehlchen/api/rest.py

```diff
@@ -4761,11 +4761,12 @@ def export_history_events(
     ) -> dict[str, Any] | Response:
         """Export or Download history events data to a CSV file."""
         dbevents = DBHistoryEvents(self.rotkehlchen.data.db)
+        has_premium = has_premium_check(self.rotkehlchen.premium)
```

Comment:
you are making an extra if check for no reason here

```suggestion
        if (has_premium := has_premium_check(self.rotkehlchen.premium)):
            filter_query = HistoryEventFilterQuery.make()
```